charts/observation-service/templates/secrets.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
```

Comment:
What kind of data does the apiConfig hold? 
Wouldn't a config Map make more sense here?

Reply:
It would contain API configurations as specified in this [file](https://github.com/caraml-dev/timber/blob/main/observation-service/config/config.go). I think there should be no issues with using a `ConfigMap`, but I think that can be a future optimization, it works as is now and this way is similarly used in [XP helm charts](https://github.com/caraml-dev/xp/tree/main/infra/charts).